apollo-runtime/src/test/java/com/apollostack/android/HeroDetails.java

```diff
@@ -0,0 +1,47 @@
+package com.apollostack.android;
+
+import com.apollostack.api.GraphQLQuery;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+public final class HeroDetails implements GraphQLQuery<GraphQLQuery.Variables> {
+  public static final String OPERATION_DEFINITION = "query HeroDetails {\n"
+      + "  allPeople {\n"
+      + "    people {\n"
+      + "      name\n"
+      + "    }\n"
+      + "  }\n"
+      + "}";
+
+  public static final String QUERY_DOCUMENT = OPERATION_DEFINITION;
+
+  private final GraphQLQuery.Variables variables;
+
+  public HeroDetails() {
+    this.variables = GraphQLQuery.EMPTY_VARIABLES;
+  }
+
+  @Override
+  public String queryDocument() {
+    return QUERY_DOCUMENT;
+  }
+
+  @Override
+  public GraphQLQuery.Variables variables() {
+    return variables;
+  }
+
+  public static class Data implements GraphQLQuery.Data {
+    @Nullable AllPeople allPeople;
+
+    static class AllPeople {
+      @Nullable List<People> people;
+
+      static class People {
```

Comment:
I've manually modified these types to be static classes instead of interfaces, so the test would pass (as described in the PR summary).
I can go ahead and change the compiler code to do this if you guys are 👍